src/plonk.rs

```diff
@@ -74,6 +77,72 @@ impl<C: CurveAffine> VerifyingKey<C> {
             cs,
         })
     }
+
+    /// Hashes a verification key into a transcript.
+    pub fn hash<T: Transcript<C>>(&self, transcript: &mut T) -> io::Result<()> {
+        let try_and_increment = |hasher: &Blake2bState| {
```

Comment:
Ah, let's just add a `common_scalar` to the `Transcript` trait, and refactor the transcript implementations to use it. Then there's no need to sample curve points.

Reply:
Added this in #179.